src/main/java/org/prebid/server/spring/config/AnalyticsConfiguration.java

```diff
@@ -302,4 +304,46 @@ private static class PubstackBufferProperties {
             Long reportTtlMs;
         }
     }
+
+    @Configuration
+    @ConditionalOnProperty(prefix = "analytics.liveintent", name = "enabled", havingValue = "true")
+    public static class LiveIntentAnalyticsConfiguration {
+
+        @Bean
+        LiveIntentAnalyticsReporter liveIntentAnalyticsReporter(
+                LiveIntentAnalyticsConfigurationProperties properties,
+                HttpClient httpClient,
+                JacksonMapper jacksonMapper
+        ) {
+            return new LiveIntentAnalyticsReporter(
+                    properties.toComponentProperties(),
+                    httpClient,
+                    jacksonMapper
+            );
+        }
+
+        @Bean
+        @ConfigurationProperties(prefix = "analytics.liveintent")
+        LiveIntentAnalyticsConfigurationProperties liveIntentAnalyticsConfigurationProperties() {
+            return new LiveIntentAnalyticsConfigurationProperties();
+        }
+
+        @Validated
+        @NoArgsConstructor
+        @Data
+        private static class LiveIntentAnalyticsConfigurationProperties {
+
+            String partnerId;
+            String analyticsEndpoint;
+            long timeoutMs;
```

Comment:
Add empty lines between properies

Reply:
Done in https://github.com/prebid/prebid-server-java/pull/4154/commits/58fc8c0e317de0aa062fddda86b40ee1758acca6